packages/components/src/components/data-list/data-list-row.jsx

```diff
@@ -23,7 +23,7 @@ const DataListRow = ({
                 measure?.();
             }
         });
-    }, [show_desc, isMounted, is_dynamic_height, measure]);
+    }, [show_desc, is_dynamic_height, measure]);
```

Comment:
suggested to add below comment to prevent eslint dependency warning.
```suggestion
        // eslint-disable-next-line react-hooks/exhaustive-deps
    }, [show_desc, is_dynamic_height, measure]);
```

Reply:
@mahdiyeh-fs Thank you so much for the suggestion, however here it doesn't give any error. Maybe it would give us such a warning if isMounted was not an extra dependency here but it is. 🙂